src/lib/mongoBackend/mongoQueryTypes.cpp

```diff
@@ -655,6 +655,7 @@ HttpStatusCode mongoEntityTypes
       *totalTypesP = docs;
       char detailsMsg[256];
       snprintf(detailsMsg, sizeof(detailsMsg), "Count: %u", *totalTypesP);
+
```

Comment:
Please keep tidy files not affected by the PR.

```suggestion
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/397ae9e496a9ce542907b10016aa464b69ba71a3